apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.jsx

```diff
@@ -247,14 +247,16 @@ const OpenManagerBtn = ({
 const OpenOnboardingBtn = () => {
   const { setDrawer } = useContext(context);
   const dispatch = useDispatch();
+  const history = useHistory();
 
   const onClick = useCallback(() => {
     setTrackingSource("device action open onboarding button");
     dispatch(setNotSeededDeviceRelaunch(true));
     dispatch(relaunchOnboarding(true));
```

Comment:
I think the problem is from this action, seems to me it's supposed to push history by itself, your fix probably works from an user standpoint but it doesn't solve the original problem

Reply:
Ok discussed on slack, closing